spyderlib/config.py

```diff
@@ -530,7 +560,9 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Meta+S',
               'editor/print': "Ctrl+P",
+              'editor/close file': 'Ctrl+W',

```

Comment:
The idea of not making this shortcut configurable is to use the same shortcut in all plugins to close tabs.